hw_07/AssemblyOne/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace AssemblyOne {
+    class Program {
+        static void Main(string[] args) {
+            Console.WriteLine("Hello World!");
```

Comment:
here you need to demonstrate all possible modifiers are set up in Motorcycle, Chopper types